easybuild/easyconfigs/s/SuiteSparse/UMFPACK_GNUMakefile-4.4.5.patch

```diff
@@ -0,0 +1,57 @@
+--- SuiteSparse/UMFPACK/Lib/GNUmakefile.org	2014-03-21 20:16:16.000000000 +0100
++++ SuiteSparse/UMFPACK/Lib/GNUmakefile	2015-10-12 16:34:54.286168878 +0200
+@@ -2,18 +2,22 @@
+ # UMFPACK Makefile for compiling on Unix systems (for GNU Make)
+ #-------------------------------------------------------------------------------
+ 
+-default: libumfpack.a
++default: libumfpack.a libumfpack.so
+ 
+ include ../../SuiteSparse_config/SuiteSparse_config.mk
+ 
+ # UMFPACK can use CHOLMOD by default as an ordering option
+ ifneq (,$(findstring -DNCHOLMOD, $(UMFPACK_CONFIG)))
+-    I = -I../Include -I../Source -I../../AMD/Include -I../../SuiteSparse_config
++   I = -I../Include -I../Source -I../../AMD/Include -I../../SuiteSparse_config
+ else
+-    I = -I../Include -I../Source -I../../AMD/Include \
+-        -I../../SuiteSparse_config -I../../CHOLMOD/Include
++   I = -I../Include -I../Source -I../../AMD/Include \
++       -I../../SuiteSparse_config -I../../CHOLMOD/Include
+ endif
+ 
++# I = -I../Include -I../Source -I../../AMD/Include -I../../SuiteSparse_config \
++#     -I../../CCOLAMD/Include -I../../CAMD/Include -I../../CHOLMOD/Include \
++#     -I$(EBROOTPARMETIS)/include -I../../COLAMD/Include    
```

Comment:
this is useless, so remove it (which implies recreating the patch, unless you're very careful in editing the patch :))